plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/PluginUtil.kt

```diff
@@ -23,8 +23,16 @@ import org.gradle.language.base.plugins.LifecycleBasePlugin
 import org.gradle.util.GradleVersion
 import java.nio.file.Path
 
+// The latest commit on ktlint's master branch is automatically uploaded to Sonatype's snapshots repository
+// https://github.com/pinterest/ktlint#access-to-the-latest-master-snapshot
```

Comment:
Can you turn these two lines into a kotlin doc comment instead?

Reply:
Done in 24d2032